Enterprise 1.11.6 release notes

gunnaraasen opened the pull request:
Add release notes for the Enterprise 1.11.6.

_Describe your proposed changes here._

- [x] Rebased/mergeable

jstirnaman commented on content/enterprise_influxdb/v1/about-the-project/release-notes.md:
```diff
@@ -19,12 +19,42 @@ InfluxDB Enterprise builds are available. For more information, see
+- Prevent retention service from hanging, which could lead to a failure to
+  remove old shards.
+- Fix shard precreation service to pre-create shards for more than the first
+  databse.
```
```suggestion
  database.
```

jstirnaman commented on content/enterprise_influxdb/v1/about-the-project/release-notes.md:
```diff
@@ -19,12 +19,42 @@ InfluxDB Enterprise builds are available. For more information, see
+- Fix shard precreation service to pre-create shards for more than the first
+  databse.
+- Fix issues with remote iterators caused by floating point literals being
+  rounded in a way that causes lose of precision and can drastically change the
```
```suggestion
  rounded in a way that causes precision loss and can drastically change the
```

jstirnaman reviewed (approved):
Typo fixes, otherwise 👍

davidby-influx reviewed (changes requested):
LGTM with Jason's issues addressed.

gunnaraasen commented:
I fixed the typos and switched the bullets to inline instead of wrap (to match previous release notes).

I've also added one last note about improved TSM error handling.

davidby-influx reviewed (approved):
LGTM
